### ISSUE TITLE
Netflix

### ISSUE THREAD

**Issue by Chippel** [ghacksuserjs/ghacks-user.js#854]
So I started using the users.js which is pretty nice for hardening. The problem I have is that I can't watch Netflix and I don't know which setting I need to change in order to watch it. Could you help me please? :)

**Comment by Thorin-Oakenpants** [comment]
did you try the `setup` tags

```js
/* 1825: disable widevine CDM (Content Decryption Module)
 * [SETUP-WEB] if you *need* CDM, e.g. Netflix, Amazon Prime, Hulu, whatever ***/
user_pref("media.gmp-widevinecdm.visible", false);
user_pref("media.gmp-widevinecdm.enabled", false);
```

**Comment by crssi** [comment]
and maybe also
``` js
/* 1830: disable all DRM content (EME: Encryption Media Extension)
 * [SETUP-WEB] if you *need* EME, e.g. Netflix, Amazon Prime, Hulu, whatever
 * [SETTING] General>DRM Content>Play DRM-controlled content
 * [1] https://www.eff.org/deeplinks/2017/10/drms-dead-canary-how-we-just-lost-web-what-we-learned-it-and-what-we-need-do-next ***/
user_pref("media.eme.enabled", false);
```

**Comment by Chippel** [comment]
Thanks for help. I've set both to true but it still doesn't work.

**Comment by crssi** [comment]
I just got a Netflix account from a friend of mine for test. Thanks @BC ;)

As @Thorin-Oakenpants  already sad... you need:
``` js
user_pref("media.gmp-widevinecdm.visible", true);
user_pref("media.gmp-widevinecdm.enabled", true);
``` 

Which is enough, but every new session you need to allow DRM which shows in the yellow bar on top of the page and every first time play it fails, the second time and on... its OK.

To overcome this annoyance, you need also:
``` js
user_pref("media.eme.enabled", true);
``` 

And that it is... if still doesn't work, then the problem lies in your overrides or some web extension(s).

Cheers

**Comment by Thorin-Oakenpants** [comment]
closing due to inactivity, and lack of information

- what is the error - on the webpage, from Firefox, in the console?
- has OP tried a new profile and confirmed Netflix works?
   - probably not necessary: millions of FF users use netflix
- has OP confirmed a new profile with ghacks user.js causes the issue
   - probably not necessary: we already know `1825` and `1830` block netflix
- has OP tried a new profile with ghacks user.js with `1825` + `1830` reset/overriden
   - probably not necessary now, as @crssi did it for him, and that this is the cause

Does OP even have widevine files in his profile: e.g. it could be that hosts or a VPN is blocking the files getting downloaded (see #825 ) .. or it could be a pref not in our user.js (e.g. some users like to blank all the urls in prefs due to bad advise elsewhere) .. or it could be an extension .. or it could be OP's host file or something

tl;dr - use the [troubleshooting wiki](https://github.com/ghacksuserjs/ghacks-user.js/wiki/1.4-Troubleshooting)
